tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -409,8 +409,8 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
     filter = str => !str.contains("newMain"),
     compat = Map(
       "3" ->
-        """|case head :: next => scala.collection.immutable
-           |case Nil => scala.collection.immutable""".stripMargin
+        """|head :: next scala.collection.immutable
+           |Nil scala.collection.immutable""".stripMargin
```

Comment:
Shouldn't these result be filtered out by fuzzy-match?

Reply:
There is a problem, because `new` is not seen on path (the tree looks the same as for `case@@`. I'm looking into it, if I won't find a solution soon I will create an issue

Reply:
fuzzy-match isn't working, because compiler doesn't see the Ident("new")

Reply:
I've created an issue https://github.com/scalameta/metals/issues/4367